server/cluster/unsafe_recovery_controller.go

```diff
@@ -80,7 +80,7 @@ func (u *unsafeRecoveryController) RemoveFailedStores(failedStores map[uint64]st
 		return errors.Errorf("Another request is working in progress")
 	}
 	u.reset()
-	for failedStore, _ := range failedStores {
+	for failedStore := range failedStores {
```

Comment:
make sure all stores are legal and then remove them

Reply:
Done. PTAL.